yarn.lock

```diff
@@ -8668,6 +8668,14 @@ dom-helpers@^3.2.1, dom-helpers@^3.3.1, dom-helpers@^3.4.0:
   dependencies:
     "@babel/runtime" "^7.1.2"
 
+dom-helpers@^5.0.1:
```

Comment:
Hmm do we have two versions of dom-helpers/react-transition-group now?

Reply:
Hmmm i went looking because i was sure we had used it for something. I did not find it. Because I was looking for CSSTransition. However, I did find it now, in OpenTransition. I'll open a PR to dedupe